docs/components/connectors/out-of-the-box-connectors/rest.md

```diff
@@ -66,6 +66,31 @@ Select the **REST Connector** and fill out the following properties under the **
 
 ![rest Connector bearer token auth](../img/connectors-rest-bearer-token-auth.png)
 
+### REST Connector (OAuth token)
+
+#### Create a new Connector secret
+
+We advise you to keep your **OAUTH_TOKEN_ENDPOINT** safe and avoid exposing it in the BPMN `xml` file by creating a secret:
+
+1. Follow our [guide for creating secrets](../../console/manage-clusters/manage-secrets.md).
+2. Name your secret (i.e `OAUTH_TOKEN_ENDPOINT`) so you can reference it later in the Connector.
+
+#### Configure the OAuth Token
+
+Select the **REST Connector** and fill out the following properties under the **Authentication** section:
+
+1. Click **OAuth 2.0** in the **Authentication** section.
+2. Set **OAuth Token Endpoint** to the secret you created (i.e. `secrets.OAUTH_TOKEN_ENDPOINT`).
+3. Set **Client ID** to the secret you created (i.e. `secrets.CLIENT_ID`).
+4. Set **Client secret** to the secret you created (i.e. `secrets.CLIENT_SECRET`).
+5. Set **Scopes** (i.e. `read:clients`). It is an optional field. Depending on the OAuth client you're using, you should fill this field or not.
```

Comment:
"Depending on the OAuth client you're using, you should fill this field or not." -- What do you mean by this? Is this optional based on the OAuth client?

Reply:
Yes, some OAuth providers don't ask for the scope and the audience.

Reply:
Great -- I adjusted the wording in my latest commit.

Reply:
Great! Thank you! If there is nothing else, can we do the merge?

Reply:
If @MoAtCamunda @Oleksiivanov and @tmetzke don't have any further comments, yes!

Reply:
Well, is the backend for this merged and released already? If not, we cannot merge this. Otherwise, the docs will be public before the functionality is.